Documentation/install/vmware/vmware-terraform.md

```diff
@@ -6,16 +6,17 @@ Generally, the VMware platform templates adhere to the standards defined by the
 
 ## Prerequsities
 
-1. Download the latest Container Linux Stable OVA from; https://coreos.com/os/docs/latest/booting-on-vmware.html.
-1. Import `coreos_production_vmware_ova.ova` into vCenter. Generally, all settings can be kept as is. Consider "thin" provisioning and naming the template with CoreOS Container Linux Version.
-1. Resize the Virtual Machine Disk size to 30 GB or larger
-1. In the Virtual Machine Configuration View select "vApp Options" tab and un-check "Enable vApp Options".
+1. Download the latest Container Linux Stable OVA from  https://coreos.com/os/docs/latest/booting-on-vmware.html.
```

Comment:
Consider making this an actual link.
Consider numbering correctly rather than 1, 1, 1...

Reply:
ok to both. i thought 1. 1. 1. was markdown standard - it automatically generates the correct numbers. no?